src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -350,6 +350,7 @@ private void RecursivelyEnumerateFiles(string folderPath, ref List<string> found
                     this.logger.WriteHandledException(
                         $"Could not enumerate files in the path '{folderPath}' due to an exception",
                         e);
+                    continue;
```

Comment:
Minor nit, all the other `continue` usages above are preceded by a blank line.  Ditto the one below on line 401.